build-scripts/rcm-guest/publish-oc-binary.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/sh
+#!/bin/bash
 set -eux
 
 rpm_name() {
     printf "${RPM}" "${arch}"
-    [ "${arch}" == x86_64 ] && printf %s -redistributable
+    [[ "${arch}" == x86_64 ]] && printf %s -redistributable
```

Comment:
nit: writing this as 

```sh
if [[ "${arch}" == x86_64 ]]; then
    printf %s -redistributable
fi
```

Makes this look like "normal" code you would see in another language (except Ruby, but whatever) and IMHO improves readability/lowers the context switching overhead for a dev to read this script.

Also of note, explicit `if` and `else` will ensure devs are not using the `test && someting || else` construct, which many thing will run like a ternary operator but in reality has a different control flow.

Anyway, just a nit. Up to you @bbguimaraes @jupierce if you want to bother with that.